spec/rubocop/target_ruby_spec.rb

```diff
@@ -269,6 +269,19 @@
           expect(target_ruby.version).to eq default_version
         end
       end
+
+      context 'when file contains a syntax error' do
+        it 'uses the default target ruby version' do
+          content = <<~HEREDOC
+            Gem::Specification.new do |s|
+              s.required_ruby_version = '>= 3.3.0''
```

Comment:
Can you add a comment to make it easier to understand?

```suggestion
              s.required_ruby_version = '>= 3.3.0'' # invalid syntax
```